File: app/src/main/java/com/nanjing/tqlhl/model/bean/MjAqiBean.java
package com.nanjing.tqlhl.model.bean;

/**
 * @author wujinming QQ:1245074510
 * @name WeatherOne
 * @class name：com.example.tianqi.model.bean
 * @class describe
 * @time 2020/9/8 15:34
 * @class describe
 */
public class MjAqiBean {
    /**
     * code : 0
     * data : {"aqi":{"cityName":"东城区","co":"4","no2":"21","o3":"13","pm10":"43","pm25":"5","pubtime":"1472688000000","rank":"222/621","so2":"1","value":"43"},"city":{"cityId":284609,"counname":"中国","name":"东城区","pname":"北京市"}}
     * msg : success
     * rc : {"c":0,"p":"success"}
     */

    private int code;
    private DataBean data;
    private String msg;
    private RcBean rc;

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public DataBean getData() {
        return data;
    }

    public void setData(DataBean data) {
        this.data = data;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public RcBean getRc() {
        return rc;
    }

    public void setRc(RcBean rc) {
        this.rc = rc;
    }

    public static class DataBean {
        /**
         * aqi : {"cityName":"东城区","co":"4","no2":"21","o3":"13","pm10":"43","pm25":"5","pubtime":"1472688000000","rank":"222/621","so2":"1","value":"43"}
         * city : {"cityId":284609,"counname":"中国","name":"东城区","pname":"北京市"}
         */

        private AqiBean aqi;
        private CityBean city;

        public AqiBean getAqi() {
            return aqi;
        }

        public void setAqi(AqiBean aqi) {
            this.aqi = aqi;
        }

        public CityBean getCity() {
            return city;
        }

        public void setCity(CityBean city) {
            this.city = city;
        }

        public static class AqiBean {
            /**
             * cityName : 东城区
             * co : 4
             * no2 : 21
             * o3 : 13
             * pm10 : 43
             * pm25 : 5
             * pubtime : 1472688000000
             * rank : 222/621
             * so2 : 1
             * value : 43
             */

            private String cityName;
            private String co;
            private String no2;
            private String o3;
            private String pm10;
            private String pm25;
            private String pubtime;
            private String rank;
            private String so2;
            private String value;

            public String getCityName() {
                return cityName;
            }

            public void setCityName(String cityName) {
                this.cityName = cityName;
            }

            public String getCo() {
                return co;
            }

            public void setCo(String co) {
                this.co = co;
            }

            public String getNo2() {
                return no2;
            }

            public void setNo2(String no2) {
                this.no2 = no2;
            }

            public String getO3() {
                return o3;
            }

            public void setO3(String o3) {
                this.o3 = o3;
            }

            public String getPm10() {
                return pm10;
            }

            public void setPm10(String pm10) {
                this.pm10 = pm10;
            }

            public String getPm25() {
                return pm25;
            }

            public void setPm25(String pm25) {
                this.pm25 = pm25;
            }

            public String getPubtime() {
                return pubtime;
            }

            public void setPubtime(String pubtime) {
                this.pubtime = pubtime;
            }

            public String getRank() {
                return rank;
            }

            public void setRank(String rank) {
                this.rank = rank;
            }

            public String getSo2() {
                return so2;
            }

            public void setSo2(String so2) {
                this.so2 = so2;
            }

            public String getValue() {
                return value;
            }

            public void setValue(String value) {
                this.value = value;
            }
        }

        public static class CityBean {
            /**
             * cityId : 284609
             * counname : 中国
             * name : 东城区
             * pname : 北京市
             */

            private int cityId;
            private String counname;
            private String name;
            private String pname;

            public int getCityId() {
                return cityId;
            }

            public void setCityId(int cityId) {
                this.cityId = cityId;
            }

            public String getCounname() {
                return counname;
            }

            public void setCounname(String counname) {
                this.counname = counname;
            }

            public String getName() {
                return name;
            }

            public void setName(String name) {
                this.name = name;
            }

            public String getPname() {
                return pname;
            }

            public void setPname(String pname) {
                this.pname = pname;
            }
        }
    }

    public static class RcBean {
        /**
         * c : 0
         * p : success
         */

        private int c;
        private String p;

        public int getC() {
            return c;
        }

        public void setC(int c) {
            this.c = c;
        }

        public String getP() {
            return p;
        }

        public void setP(String p) {
            this.p = p;
        }
    }
}

File: app/src/main/java/com/nanjing/tqlhl/model/bean/MjAqiBean.kt
package com.nanjing.tqlhl.model.bean

/**
 * @author wujinming QQ:1245074510
 * @name WeatherOne
 * @class name：com.example.tianqi.model.bean
 * @class describe
 * @time 2020/9/8 15:34
 * @class describe
 */
class MjAqiBean {
    /**
     * code : 0
     * data : {"aqi":{"cityName":"东城区","co":"4","no2":"21","o3":"13","pm10":"43","pm25":"5","pubtime":"1472688000000","rank":"222/621","so2":"1","value":"43"},"city":{"cityId":284609,"counname":"中国","name":"东城区","pname":"北京市"}}
     * msg : success
     * rc : {"c":0,"p":"success"}
     */
    var code = 0
    var data: DataBean? = null
    var msg: String? = null
    var rc: RcBean? = null

    class DataBean {
        /**
         * aqi : {"cityName":"东城区","co":"4","no2":"21","o3":"13","pm10":"43","pm25":"5","pubtime":"1472688000000","rank":"222/621","so2":"1","value":"43"}
         * city : {"cityId":284609,"counname":"中国","name":"东城区","pname":"北京市"}
         */
        var aqi: AqiBean? = null
        var city: CityBean? = null

        class AqiBean {
            /**
             * cityName : 东城区
             * co : 4
             * no2 : 21
             * o3 : 13
             * pm10 : 43
             * pm25 : 5
             * pubtime : 1472688000000
             * rank : 222/621
             * so2 : 1
             * value : 43
             */
            var cityName: String? = null
            var co: String? = null
            var no2: String? = null
            var o3: String? = null
            var pm10: String? = null
            var pm25: String? = null
            var pubtime: String? = null
            var rank: String? = null
            var so2: String? = null
            var value: String? = null
        }

        class CityBean {
            /**
             * cityId : 284609
             * counname : 中国
             * name : 东城区
             * pname : 北京市
             */
            var cityId = 0
            var counname: String? = null
            var name: String? = null
            var pname: String? = null
        }
    }

    class RcBean {
        /**
         * c : 0
         * p : success
         */
        var c = 0
        var p: String? = null
    }
}